### PR TITLE
Add instructions for Dev Env in Windows.

### PR DESCRIPTION
**Pull request by Pedro-Revez-Silva**
## Description
There were no instructions on how to setup the development environment for Windows machines.

Addresses: 
- Couldn't find any specific issues for this.

## Screenshots
Doesn't apply

### CODE REVIEW

**Comment by Pedro-Revez-Silva** [comment]
I have read the CLA Document and I hereby sign the CLA

**Review by shogunpurple — APPROVED** [review]
Thanks for putting this together @Pedro-Revez-Silva - it will help a lot of people 👌

**Comment by joebudi** [comment]
Nice work @Pedro-Revez-Silva 🙌

**Review by Mitch-Budibase — APPROVED** [review]
Great work!

File: docs/DEV-SETUP-WINDOWS.md [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+### Install NVM & Node 16
+NVM documentation: https://github.com/nvm-sh/nvm#installing-and-updating
+
+Install NVM
+```
+curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.39.0/install.sh | bash
+```
+Install Node 16
+```
+nvm install 16
```
Comment (Rory-Powell) [inline docs/DEV-SETUP-WINDOWS.md:35]: It's worth noting that the officially supported version of node for budibase is `14`. Our docker images are curently based on `node:14-slim`

Comment (Rory-Powell) [reply docs/DEV-SETUP-WINDOWS.md:35]: The application may still function completely fine on later versions, but 14 is the version we deploy and test with so there is en element of risk developing with 16+, i.e. features that exist in 16 but not in 14. The safest bet is to develop with the officially supported version.

**Pull request merged**